validation/src/main/scala/hmda/validation/rules/lar/quality/Q022.scala

```diff
@@ -11,18 +11,22 @@ import scala.util.Try
 object Q022 {
   def apply(lar: LoanApplicationRegister, year: Int): Result = {
     // This edit should not fail if applicationDate is equal to "NA" or is otherwise non-convertible
-    val applicationYear = Try(lar.loan.applicationDate.substring(0, 4).toInt).getOrElse(year)
+    val applicationYear = parseYear(lar).getOrElse(year)
```

Comment:
Why is the `getOrElse` in a different place on each method? Not sure whether I'm missing some nuance in their placement, or if they're effectively the same.